avocado/core/safeloader/core.py

```diff
@@ -11,8 +11,14 @@
 
 
 def get_methods_info(statement_body, class_tags, class_requirements):
-    """
-    Returns information on an Avocado instrumented test method
+    """Returns information on test methods.
+
+    :param statement_body: a the body of a "class" statement
```

Comment:
typo a/the

Reply:
Thanks @richtja, I'll force push this small fix.